app/event_source/subscribers/enrollment_subscriber.rb

```diff
@@ -113,7 +113,7 @@ def redetermine_family_eligibility(subscriber_logger, payload)
 
     private
 
-    def fetch_application(enrollment)
+    def fetch_application(enrollment, family)
```

Comment:
Instead of passing family, can you modify this to 
```suggestion
    def fetch_application(enrollment, family)
    enrollment.family
```

Reply:
Great suggestion. Done.